### DISCUSSION TITLE
CTB Editor Switch Between Standard View and Catch View

### DISCUSSION THREAD

**Discussion by Samakura** [ppy/osu#21513] [Ideas] [answered]
Hello, so currently in osu!lazer in the catch the beat editor you are only able to map in the catch the beat view (something that is new in lazer.) However most catch mappers (including myself) have learned to map via the standard editor shown in the screenshot below:

![screenshot025](https://user-images.githubusercontent.com/76218256/205538878-a0266499-8714-4185-acb2-a383154a465a.jpg)

With the new lazer editor it introduces a new form of mapping shown here: 

![example 2](https://user-images.githubusercontent.com/76218256/205540484-f83c0e70-2058-45a9-8651-83f187e4d293.jpg)

Its a cool new approach to catch mapping, and I'm sure people will enjoy mapping this way but for people like me and some others who have learned the traditional method of catch mapping, it might be hard to adjust. Also not being able to see the patterns via the standard view can pose to be difficult for some.

Basically my point is that a lot of people may have gotten accustomed to mapping catch the beat using the tools in the standard editor, which to my knowledge is completely taken away in the new catch editor in lazer, as it takes a new approach to catch mapping.

My suggestion would be to add a toggle that allowed the user to switch between the new catch the beat editor and the traditional standard editor, as most are familiar with to be more inclusive to these people. Maybe somewhere in this view tab as shown below:

![example](https://user-images.githubusercontent.com/76218256/205539514-5f672e39-56ce-4f89-a08a-222a5314a030.png)

**Comment by peppy** [comment, marked as answer]
This isn't technically possible. Please continue to use stable, it will continue to be supported for some years.

**Reply by Samakura** [reply]
Ah I see, appreciate the response!
